src/Integration.Vsix/Suppression/LiveIssue.cs

```diff
@@ -24,10 +24,10 @@
 namespace SonarLint.VisualStudio.Integration.Vsix.Suppression
 {
     /// <summary>
-    ///     Information about a single Roslyn issue, decorated with the extra information required to map it to a
-    ///     SonarQube issue.
+    /// Information about a single Roslyn issue, decorated with the extra information required to map it to a
+    /// SonarQube issue.
     /// </summary>
-    internal class LiveIssue
+    public class LiveIssue
```

Comment:
Why this change?

Reply:
ILiveIssueFactory needs to be public so Moq can mock it so LiveIssue needs to be public too as it's returned by the interface.